src/Faker/Provider/de_AT/Person.php

```diff
@@ -121,28 +123,33 @@ public static function suffix()
     /**
      * Generates a random Austrian Social Security number.
      * @link https://de.wikipedia.org/wiki/Sozialversicherungsnummer#.C3.96sterreich
+     * @param  \DateTime|null  $birthdate
      * @return string
      */
-    public static function ssn()
+    public static function ssn(\DateTime $birthdate = null)
```

Comment:
You can add return type hint: `: string`

Reply:
Definitely a good idea - ideally that's something that's done across the whole codebase I guess - but why not start here :)

Reply:
Since adding it here won't result in a break change, I think it will be fine.